### PR TITLE
added tika Extractor and removed tika server

### PR DESCRIPTION
**Pull request by raghuramlavan**
I have removed the tika server and has included the tika-extractor .

### CODE REVIEW

File: flake.nix [hunk 1]
```diff
@@ -44,6 +46,17 @@
       npmlock2nixOverlay = final: prev: {
         npmlock2nix = import npmlock2nix-src { pkgs = prev; };
       };
+ 
+
+      pkgsForSystem = forAllSystems( system: import nixpkgs {
+           # ./overlay.nix contains the logic to package local repository
+           overlays = [ mvn2nix.overlay (
+               final: prev: {
+               tikaExtractor = final.callPackage ./tikaExtractor.nix { };
```
Comment (happysalada) [inline flake.nix]: could you please add this in the main overlay, right where tika-server was ?

File: tikaExtractor.nix [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+  installPhase = ''
+    # create the bin directory
+    mkdir -p $out/bin
+
+    # create a symbolic link for the lib directory
+    ln -s ${mavenRepository} $out/lib
+
+    # copy out the JAR
+    ls -l
```
Comment (happysalada) [inline tikaExtractor.nix]: remaining debug statements

**Review by happysalada — COMMENTED** [review]
There are some merge conflicts with the latest commits. Would you mind resolving the conflicts ? I'm happy to merge this once the conflicts are addressed.
I have a couple comments if you have time, otherwise I can do those myself, no worries.

Thank you again for this!

**Comment by raghuramlavan** [comment]
> There are some merge conflicts with the latest commits. Would you mind resolving the conflicts ? I'm happy to merge this once the conflicts are addressed.
> I have a couple comments if you have time, otherwise I can do those myself, no worries.
> 
> Thank you again for this!

ok i will do these.

File: flake.nix (hunk 1 above)
Comment (raghuramlavan) [reply flake.nix]: i have added the same.

File: tikaExtractor.nix (hunk 2 above)
Comment (raghuramlavan) [reply tikaExtractor.nix]: They are requred to keep the jar in a standard loc as per. Iam using the template from mvn2nix

Comment (happysalada) [reply tikaExtractor.nix]: ls -l is just a way to list the files, It doesn't make copies.

**Comment by happysalada** [comment]
I'm happy to merge!

Thank you for this!

**Pull request merged**